package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "contentstack",
-  "version": "3.17.2",
```

Comment:
Since we are adding support of the classes and functions so it needs to be updated as v3.18.0

@abhinav-from-contentstack

Reply:
done